packages/node_modules/@ciscospark/plugin-conversation/src/share-activity.js

```diff
@@ -233,9 +233,9 @@ const ShareActivity = SparkPlugin.extend({
       verb: `share`,
       object: {
         objectType: `content`,
-        displayName: this.displayName,
-        content: this.content,
-        mentions: this.mentions,
+        displayName: this.object && this.object.displayName ? this.object.displayName : ``,
```

Comment:
At this point we should go with this change and may be revisit the ShareActivity later again.